examples/refresh.amp.html

```diff
@@ -6,6 +6,8 @@
     <link rel="canonical" href="http://nonblocking.io/" >
     <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
     <meta name="amp-ad-enable-refresh" content="doubleclick=30">
+    <!-- remove the next meta tag once refresh is stable -->
+    <meta name="amp-experiments-opt-in" content="amp-ad-refresh">
```

Comment:
Removed. (These changes were made along with the other PR I sent you, but that's been scrapped now.)